tests/security/grub_auth/grub_authorization.pm

```diff
@@ -31,7 +31,8 @@ my $test_passwd = 'pw_test';
 sub switch_boot_menu {
     my $switch = shift;
 
-    if (get_var('UEFI')) {
+    # UEFI item is available only from 15-SP4
+    if (get_var('UEFI') && is_sle('>15-SP3')) {
```

Comment:
https://openqa.opensuse.org/tests/2356255
https://openqa.opensuse.org/tests/2355049

Can you please also do VR on TW platforms?

Reply:
good point! Will do, thanks :)